docs/nodes/analyzer/weighted_vector_sum.rst

```diff
@@ -0,0 +1,79 @@
+Weighted Vector Sum (Alpha)
```

Comment:
I would suggest to name the node "Center of Mass - Mesh" (as opposed to existing Center of Mass node for solids; maybe that node should be renamed to "Center of Mass - Solid", for clarity).